packages/slice-machine/src/icons/CloseIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import type { FC, SVGProps } from "react";
+
+export const CloseIcon: FC<SVGProps<SVGSVGElement>> = (props) => (
+  <svg
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
```

Comment:
First two icons with viewBox 24, here. It's the viewbox of the editor, on our side Benjamin has a viewbox of 32, is that normal?